Modules/getpath.c

```diff
@@ -1016,15 +1016,21 @@ calculate_zip_path(PyCalculatePath *calculate, const wchar_t *prefix)
             return PATHLEN_ERR();
         }
     }
-    status = joinpath(calculate->zip_path, L"lib/python00.zip", zip_path_len);
+    status = joinpath(calculate->zip_path, L"lib/python000.zip", zip_path_len);
```

Comment:
I believe this change will cause backward compatibility issues.

Reply:
> I believe this change will cause backward compatibility issues.

Why do you think that? The change looks harmless to me, especially with the code below that replaces the three zeros by the version information. That code shortens the file name when the version information only requires two characters.

Reply:
> That code shortens the file name when the version information only requires two characters.

You're right. Disregard my comment.